draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -649,7 +649,7 @@
    It MUST be supported by uCDN and by dCDN.
    </t>
    <t>
-   The CI/T interface uses a root URI for retrieval of the trigger collection resource and creation of new triggers.
+   The CI/T interface uses a root URI for retrieval of the trigger collection resource and the creation of new triggers.
```

Comment:
Adding a "the" might be okay here, but it would sound better if it was before "retrieval" instead, i.e.:
"...the retrieval of the trigger collection resource and creation of new triggers."